crates/swc_ecma_parser/tests/typescript/issue-2174/case2/input.tsx

```diff
@@ -1,5 +0,0 @@
-const x = {
```

Comment:
Why did you delete this test?

Reply:
@kdy1 I commented above. This isn't valid TS syntax

> Also had to remove test for https://github.com/swc-project/swc/issues/2174. This is actually invalid syntax in TS see https://www.typescriptlang.org/play?#code/MYewdgzgLgBAHjAvDA3gKBpmAHATibALhgEsIBhEXXAU2CgyyYH4YAzMJAPhgAoUYAXwCUjJuJjEO3GFFwBXGgBo0ggNxA

Reply:
It's valid JavaScript though. This is just a failing of the TS compiler.